### DISCUSSION TITLE
"No frames have been detected check error logs" error when using frigate and home assistant.

### DISCUSSION THREAD

**Discussion by aaronjrodrigues** [blakeblackshear/frigate#14053] [Ask A Question]
I've been using frigate with homeassistant for and I've noticed that after an hour or so the frigate camera feed's thumbnail shows 'No frames have been received check error logs' and in addition to that I no longer get detect notifications. But if I click on the thumbnails I can still view the live webrtc stream. To fix the issue I gotta restart the frigate docker container and then it works again for an hour or so.

Here is my frigate config file: https://pastebin.com/shzafLNf Any help would be appreciated, thanks. In addition to that I'm running frigate docker on an ubuntu lxc container.

**Comment by NickM-27** [comment]
Please show a screenshot of the system and camera metrics while this is happening.

**Reply by aaronjrodrigues** [reply]
Aw man I just restarted the instance so now I gotta wait for it to happen again. I'll upload a screenshot in the next hour when it happens.

**Comment by aaronjrodrigues** [comment]
@NickM-27 

<img width="1460" alt="image" src="https://github.com/user-attachments/assets/634ae9cb-b596-40da-a18a-1e956e2d2c4f">
<img width="1460" alt="image" src="https://github.com/user-attachments/assets/56b6d804-ddb1-4314-8610-b68e198ad68c">

**Reply by NickM-27** [reply]
we need to see the camera metrics too, but your coral is running slowly so most likely that is the issue, your cameras are seeing too much activity for the coral to keep up. This can be due to a bad cable, slow usb port, or some slowdown like if you are running frigate in a VM
